manifests/charts/istio-control/istio-discovery/templates/configmap.yaml

```diff
@@ -1,8 +1,7 @@
-
 {{- define "mesh" }}
     # The trust domain corresponds to the trust root of a system.
     # Refer to https://github.com/spiffe/spiffe/blob/master/standards/SPIFFE-ID.md#21-trust-domain
-    trustDomain: {{ .Values.global.trustDomain | default "cluster.local" | quote }}
+    trustDomain: "cluster.local"
```

Comment:
This should be removed, we rely on the built-in defaults here

Reply:
When deploying through helm, if meshConfig.trustDomain is NEVER specified OR meshConfig itself is never specified in values.yaml, will we not need this default assignment?

Reply:
No - the code has it built-in.